tests/Propel/Tests/Generator/Behavior/Sluggable/SluggableBehaviorTest.php

```diff
@@ -111,7 +111,12 @@ public static function cleanupSlugProvider()
     public function testObjectCleanupSlugPart($in, $out)
     {
         $t = new TestableTable13();
+        // locale must be utf8 for be sure test work (iconv)
+        $locale = setlocale(LC_CTYPE, null);
+        setlocale(LC_CTYPE, 'en_US.utf8');
         $this->assertEquals($out, $t->cleanupSlugPart($in), 'cleanupSlugPart() cleans up the slug part');
+        // restore to old
+        setlocale(LC_CTYPE, $locale);
```

Comment:
This needs to be place in the tearDown method as assertEquals can fail and will stop the method.